woocommerce/woocommerce-gutenberg-products-block
# Product gallery/dialog/enhancement

## What

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/11717

This PR contains small fixes/enhancements to the dialog.

- Hide the pager in dialog by default.
- Add the product title in the dialog as proposed in the design.
- Default the dialog gallery to wide width.
- Reset the main gallery image when the dialog closes.

## Why



With these small changes, we're getting closer to the proposed design and better UX.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Go to Appearance->Editor->Templates->Single Product.
2. Add the Product Gallery Block to the template.
3. Go to a product with 5+ product images in the frontend.
4. Click on the main image to open the dialog.
5. Ensure you see the product title on top left.
6. Ensure the pager navigation is hidden.
7. Click on the next image button until you reached the last image.
8. Close the dialog and ensure the main product image is now back to the original featured main image.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
